### PR TITLE
Add India district level data.

### PR DESCRIPTION
**Pull request by themonk911**
Remove IN_DD (Daman and Diu) from L2, since it is no longer a Union Territory
as of 2020. It is now included as a district under IN_DN.

### CODE REVIEW

File: src/data/metadata.csv [hunk 1]
```diff
@@ -9053,6 +9052,741 @@ IN_TR,IN,India,TR,Tripura,,,,,,
 IN_UP,IN,India,UP,Uttar Pradesh,,,,,,
 IN_UT,IN,India,UT,Uttarakhand,,,,,,
 IN_WB,IN,India,WB,West Bengal,,,,,,
+IN_AN_NMA,IN,India,AN,Andaman and Nicobar Islands,NMA,North and Middle Andaman,,,North and Middle Andaman,
```
Comment (owahltinez) [inline src/data/metadata.csv:9055]: Please sort these lexicographically (i.e. the order should be `IN_AN`, `IN_AN_NMA`, ..., `IN_AP`, `IN_AP_ANT`, ...)

File: src/data/metadata.csv [hunk 2]
```diff
@@ -9053,6 +9052,741 @@ IN_TR,IN,India,TR,Tripura,,,,,,
...
+IN_AN_NIC,IN,India,AN,Andaman and Nicobar Islands,NIC,Nicobars,,,Nicobars,
```
Comment (owahltinez) [inline src/data/metadata.csv:9056]: No need for a `match_string` if it's the same as `subregion2_name`

File: src/data/metadata.csv [hunk 3]
```diff
@@ -9023,9 +9023,8 @@ IN_AS,IN,India,AS,Assam,,,,,,
 IN_BR,IN,India,BR,Bihar,,,,,,
 IN_CH,IN,India,CH,Chandigarh,,,,,,
 IN_CT,IN,India,CT,Chhattisgarh,,,,,,
-IN_DD,IN,India,DD,Daman and Diu,,,,,,
```
Comment (owahltinez) [inline src/data/metadata.csv:9026]: Were we not getting data from this particular subregion?

Comment (themonk911) [reply src/data/metadata.csv:9026]: Re-added because mobility reports are reported for this region.

File: src/data/metadata.csv (hunk 1 above)
Comment (themonk911) [reply src/data/metadata.csv:9055]: Done.

File: src/data/metadata.csv (hunk 2 above)
Comment (themonk911) [reply src/data/metadata.csv:9056]: Done.

**Pull request merged**